clients/src/main/java/org/apache/kafka/common/requests/ConsumerGroupDescribeRequest.java

```diff
@@ -83,4 +85,16 @@ public static ConsumerGroupDescribeRequest parse(ByteBuffer buffer, short versio
             version
         );
     }
+
+    public static List<ConsumerGroupDescribeResponseData.DescribedGroup> getErrorDescribedGroupList(
+        List<String> groupIds,
+        Errors error
+    ) {
+        return groupIds.stream()
+            .map(groupId -> new ConsumerGroupDescribeResponseData.DescribedGroup()
+                .setGroupId(groupId)
+                .setErrorCode(error.code())
+                .setErrorMessage(error.message())
```

Comment:
It would be better to remove the message here. We usually don't put it if it is the default one.